pallets/xvm/src/weights.rs

```diff
@@ -59,15 +59,15 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 		// Proof Size summary in bytes:
 		//  Measured:  `0`
 		//  Estimated: `0`
-		// Minimum execution time: 771_000 picoseconds.
-		Weight::from_parts(818_000, 0)
+		// Minimum execution time: 730_000 picoseconds.
+		Weight::from_parts(754_000, 0)
 	}
 	fn wasm_call_overheads() -> Weight {
 		// Proof Size summary in bytes:
 		//  Measured:  `0`
 		//  Estimated: `0`
-		// Minimum execution time: 304_000 picoseconds.
-		Weight::from_parts(337_000, 0)
+		// Minimum execution time: 309_000 picoseconds.
+		Weight::from_parts(347_000, 0)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The minimum execution time value in the `wasm_call_overheads` function has changed. Please ensure that the new value of `309_000 picoseconds` is accurate.
```diff
-		// Minimum execution time: 304_000 picoseconds.
-		Weight::from_parts(337_000, 0)
+		// Minimum execution time: 309_000 picoseconds.
+		Weight::from_parts(347_000, 0)
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->